osu.Game/Skinning/SkinConfiguration.cs

```diff
@@ -16,5 +16,7 @@ public class SkinConfiguration : IHasComboColours, IHasCustomColours
         public Dictionary<string, Color4> CustomColours { get; set; } = new Dictionary<string, Color4>();
 
         public string HitCircleFont { get; set; } = "default";
+
+        public bool? CursorExpand { get; set; } = true;
```

Comment:
any reason for making this nullable?

Reply:
https://github.com/ppy/osu/pull/3837#discussion_r239968180
https://github.com/ppy/osu/pull/3837#discussion_r239968880